cmd/admin-handlers.go

```diff
@@ -330,6 +350,40 @@ func (a adminAPIHandlers) PerfInfoHandler(w http.ResponseWriter, r *http.Request
 		// Reply with performance information (across nodes in a
 		// distributed setup) as json.
 		writeSuccessResponseJSON(w, jsonBytes)
+	} else if perfType == "cpu" {
+		// Get CPU performance details from local server's cpu(s)
+		cpu := localEndpointsCPUPerf(globalEndpoints)
+		// Notify all other Minio peers to report cpu utilization numbers
+		cpus := globalNotificationSys.CPUPerfInfo()
+		cpus = append(cpus, cpu)
+
+		// Marshal API response
+		jsonBytes, err := json.Marshal(cpus)
+		if err != nil {
+			writeErrorResponseJSON(w, toAdminAPIErrCode(ctx, err), r.URL)
+			return
+		}
+
+		// Reply with performance information (across nodes in a
+		// distributed setup) as json.
+		writeSuccessResponseJSON(w, jsonBytes)
```

Comment:
For CPU i think the idiomatic term is `utilisation` instead of performance. Variables / comments can be changed accordingly